packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -99,6 +116,10 @@ const MarketPlacePage = () => {
     );
   }
 
+  const searchResults = matchSearch(marketplacePluginsResponse.data, searchQuery);
+  const displayedPlugins =
+    searchResults.length && searchQuery.length ? searchResults : marketplacePluginsResponse.data;
```

Comment:
I don't think this is needed you can directly use the `searchResults` here `match-sorter` handles this case directly.

Reply:
We are fetching the plugins from airtable sorted alphabetically, It seems `match-sorter` breaks this with its own sorting.

Reply:
It does not seems to be changing the sorting.